docs/book/README.md

```diff
@@ -1,6 +1,6 @@
 # Kubernetes vSphere CSI Driver
 
-vSphere Storage for Kubernetes, also called vSphere Cloud Provider, was introduced in 2017 and became the first vSphere storage solution for Kubernetes. The main goal of that project was to expose vSphere storage and features to Kubernetes users. The project offered an in-tree volume driver that has been actively used by various Kubernetes as a service solutions, such as PKS, OpenShift, GKE On-Prem, and so on. Cloud Native Storage (CNS) is a result of evolution and productization of vSphere Storage for Kubernetes and is also enterprise ready.
+vSphere Storage for Kubernetes, also called vSphere Cloud Provider, was introduced in 2017 and became the first vSphere storage solution for Kubernetes. The main goal of that project was to expose vSphere storage and features to Kubernetes users. The project offered an in-tree volume driver that has been actively used by various Kubernetes distributors such as PKS, OpenShift, GKE On-Prem etc as a service solution. Cloud Native Storage (CNS) is a result of evolution and productization of vSphere Storage for Kubernetes and is also enterprise ready.
```

Comment:
I think the original intent of this sentence was "Kubernetes As A Service" (KaaS?). Replacing that with "distributions" sounds okay to me, but then we wouldn't need the "as a service solution" part at the end of this sentence.